### PR TITLE
[API compatibility] Update 13 API Docs for Parameter Aliases

### PR DESCRIPTION
**Pull request by DongBaiYue**
以下API接口进行装饰器参数适配后，需要修改对应的中文文档说明。
paddle.concat
paddle.empty_like
paddle.full
paddle.nn.functional.softmax
paddle.nonzero
paddle.ones
paddle.outer
paddle.pow
paddle.randn
paddle.reshape
paddle.where
paddle.zeros_like
paddle.unbind

### CODE REVIEW

**Review by zhangbo9674 — APPROVED** [review]

File: docs/api/paddle/nn/functional/softmax_cn.rst
Comment (mattheliu) [inline docs/api/paddle/nn/functional/softmax_cn.rst:1]: paddle.nn.functional.softmax预览链接里没有构建成功。 http://preview-pr-7389.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/Softmax_cn.html

**Comment by mattheliu** [comment]
LGTM

**Review by mattheliu — APPROVED** [review]
LGTM

File: docs/api/paddle/nn/functional/softmax_cn.rst
Comment (DongBaiYue) [reply docs/api/paddle/nn/functional/softmax_cn.rst:1]: paddle.nn.functional.softmax对应的页面是这个http://preview-pr-7389.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/functional/softmax_cn.html

**Review by sunzhongkai588 — APPROVED** [review]

File: docs/api/paddle/nn/functional/softmax_cn.rst [hunk 1]
```diff
@@ -80,12 +80,14 @@ softmax
...
 
+.. note::
+    别名支持: 参数名 ``input`` 可替代 ``x``，参数名 ``dim`` 可替代 ``axis`` ，如 ``softmax(input=tensor_x, dim=1, ...)`` 等价于 ``softmax(x=tensor_x, axis=1, ...)`` 。
 
 参数
 ::::::::::
 
-    - **x** (Tensor) - 输入的 ``Tensor``，数据类型为 bfloat16 、 float16 、 float32 或 float64。
-    - **axis** (int，可选) - 指定对输入 :attr:`x` 进行运算的轴。:attr:`axis` 的有效范围是 :math:`[-D, D)`，:math:`D` 是输入 :attr:`x` 的维度，:attr:`axis` 为负值时与 :math:`axis + D` 等价。默认值为 -1。
+    - **x** (Tensor) - 输入的 ``Tensor``，数据类型为 bfloat16 、 float16 、 float32 或 float64。别名:： ``input``。
```
Comment (sunzhongkai588) [inline docs/api/paddle/nn/functional/softmax_cn.rst]:
```suggestion
    - **x** (Tensor) - 输入的 ``Tensor``，数据类型为 bfloat16 、 float16 、 float32 或 float64。别名： ``input``。
```

File: docs/api/paddle/nonzero_cn.rst [hunk 2]
```diff
@@ -12,11 +12,15 @@ nonzero
...
+.. note::
+    别名支持: 参数名 ``input`` 可替代 ``x`` ，如 ``nonzero(input=tensor_x)`` 等价于 ``nonzero(x=tensor_x)`` 。
+
 参数
 :::::::::
 
-    - **x** （Tensor）– 输入的 Tensor。
+    - **x** （Tensor）– 输入的 Tensor。别名： ``input``。
     - **as_tuple** (bool，可选) - 返回格式。是否以 ``1-D Tensor`` 构成的元组格式返回。
+    - **out** (Tensor，可选) - 输出 Tensor。
```
Comment (sunzhongkai588) [inline docs/api/paddle/nonzero_cn.rst]: 写清楚默认值，强调一下是关键字参数

File: docs/api/paddle/randn_cn.rst [hunk 3]
```diff
@@ -9,7 +9,7 @@ randn
 
 参数
 ::::::::::
-  - **shape** (list|tuple|Tensor) - 生成的随机 Tensor 的形状。如果 ``shape`` 是 list、tuple，则其中的元素可以是 int，或者是形状为[]且数据类型为 int32、int64 的 0-D Tensor。如果 ``shape`` 是 Tensor，则是数据类型为 int32、int64 的 1-D Tensor。
+  - **shape** (list|tuple|Tensor|*shape) - 生成的随机 Tensor 的形状。如果 ``shape`` 是 list、tuple，则其中的元素可以是 int，或者是形状为[]且数据类型为 int32、int64 的 0-D Tensor。如果 ``shape`` 是 Tensor，则是数据类型为 int32、int64 的 1-D Tensor。 如果是 ``*shape``，则直接以可变长参数形式传入整数（例如 ``randn(2, 3)``）。别名： ``size``。
```
Comment (sunzhongkai588) [inline docs/api/paddle/randn_cn.rst]:
```suggestion
  - **shape** (list|tuple|Tensor|*shape) - 生成的随机 Tensor 的形状。如果 ``shape`` 是 list、tuple，则其中的元素可以是 int，或者是形状为[]且数据类型为 int32、int64 的 0-D Tensor。如果 ``shape`` 是 Tensor，则是数据类型为 int32、int64 的 1-D Tensor。 如果是 ``shape`` 是 \*shape，则直接以可变长参数形式传入整数（例如 ``randn(2, 3)``）。别名： ``size``。
```

File: docs/api/paddle/reshape_cn.rst [hunk 4]
```diff
@@ -28,11 +28,14 @@ reshape
...
+.. note::
+    别名支持: 参数名 ``input`` 可替代 ``x``，如 ``reshape(input=tensor_x, ...)`` 等价于 ``reshape(x=tensor_x, ...)``。
+
 参数
 ::::::::::::
 
-  - **x** (Tensor) - N-D ``Tensor``，数据类型为 ``float16``, ``float32``, ``float64``, ``int16``, ``int32``, ``int64``, ``int8``, ``uint8``, ``complex64``, ``complex128``, ``bfloat16`` 或 ``bool``。
-  - **shape** (list|tuple|Tensor) - 数据类型是 ``int32``。定义目标形状。目标形状最多只能有一个维度为-1。如果 ``shape`` 的类型是 list 或 tuple，它的元素可以是整数或者形状为[]的 ``Tensor``。如果 ``shape`` 的类型是 ``Tensor``，则是 1-D 的 ``Tensor``。
+  - **x** (Tensor) - N-D ``Tensor``，数据类型为 ``float16``, ``float32``, ``float64``, ``int16``, ``int32``, ``int64``, ``int8``, ``uint8``, ``complex64``, ``complex128``, ``bfloat16`` 或 ``bool``。别名： ``input``。
+  - **shape** (list|tuple|Tensor|*shape) - 数据类型是 ``int32``。定义目标形状。目标形状最多只能有一个维度为-1。如果 ``shape`` 的类型是 list 或 tuple，它的元素可以是整数或者形状为[]的 ``Tensor``。如果 ``shape`` 的类型是 ``Tensor``，则是 1-D 的 ``Tensor``。 如果是 ``*shape``，则直接以可变长参数形式传入整数（例如 ``reshape(tensor_x, 2, 3)``）。
```
Comment (sunzhongkai588) [inline docs/api/paddle/reshape_cn.rst]:
```suggestion
  - **shape** (list|tuple|Tensor|*shape) - 数据类型是 ``int32``。定义目标形状。目标形状最多只能有一个维度为-1。如果 ``shape`` 的类型是 list 或 tuple，它的元素可以是整数或者形状为[]的 ``Tensor``。如果 ``shape`` 的类型是 ``Tensor``，则是 1-D 的 ``Tensor``。 如果``shape``是 ``*shape``，则直接以可变长参数形式传入整数（例如 ``reshape(tensor_x, 2, 3)``）。
```

**Review by sunzhongkai588 — APPROVED** [review]
LGTM
